### ISSUE TITLE
Aggregate Table 9 - All reports showing just 0s

### ISSUE THREAD

**Issue by keileyo** [cfpb/hmda-platform#1641]
The calculations for **Aggregate Table 9**  "DISPOSITION OF LOAN APPLICATIONS, BY MEDIAN AGE OF HOMES IN CENSUS TRACT IN WHICH PROPERTY IS LOCATED AND TYPE OF LOAN" are not working properly. Aggregate table values are showing results of zero throughout the tables. 

Ex: https://ffiec.cfpb.gov/data-publication/aggregate-reports/2017/MA/14454/9

Table 9 ticket is here: https://github.com/cfpb/hmda-platform/issues/1203

**Comment by keileyo** [comment]
I will update this ticket with what the values should be for a portion of the example report above. We should also look into whether the top level filtering is the issue. It should be "Loans on 1- to 4-Family **and** Manufactured Home Dwellings." Ours shows "Loans on 1- to 4-Family Manufactured Home Dwellings" so it may be looking only for manufactured homes. 

![image](https://user-images.githubusercontent.com/13036571/41101298-c87b3cae-6a31-11e8-8bd0-95a2aeba79e8.png)

**Comment by wpears** [comment]
@keileyo That label comes from the front-end, should there be an "and" included in tables 1 and 2 as well?

**Comment by colleenms** [comment]
Yes, it should be updated in tables 1/2 as well @wpears @keileyo
